src/client/styles/base.scss

```diff
@@ -1,5 +1,2 @@
-@import './variables.scss';
-
-body {
-  background-color: $global-bg;
-}
+@import './variables';
+@import "nhsuk-frontend/dist/scss/nhsuk";
```

Comment:
added in https://github.com/nhsuk/register-with-a-gp-beta-web/pull/1/commits/9d28cfab1b48aa61f46bc2e10fa4150e4c108eab